src/main/resources/rules/deployment/credibility_mapping/accumelate.json

```diff
@@ -5,7 +5,7 @@
     "priority": 10,
     "condition": "true",
     "actions": [
-      "final_credibility = Collections.min(module_credibility.values());"
```

Comment:
The MVEL could not recognize Collections even though I added java.util.Collections it to the parser context. No clue why it didn't.

Reply:
have you extended it from callback interface?

Reply:
Yes, I have made a PolicyEngineCallback class that implements the callback interface and defines:
public Map<String, Credibilty> getModuleCredibility();
public Credibility getFinalCredibility();
public void setFinalCredibility(Credibility credibility);